src/main/java/icbm/classic/api/events/BlastBlockModifyEvent.java

```diff
@@ -0,0 +1,93 @@
+package icbm.classic.api.events;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.eventhandler.Event;
+
+public class BlastBlockModifyEvent extends Event
+{
+    private World world = null;
+    private BlockPos position = null;
+    private IBlockState newState = null;
+    private int flags = 0;
+    private Runnable callback = null;
+    private boolean placedBackDown = false;
+
+    public enum BlockBreakType
```

Comment:
I rather not have nested classes in my APIs

Reply:
I can go ahead and pull `BlockBreakType` out of `BlastBlockModifyEvent` (as well as renaming it so that it's still obvious what it is meant to be used with), however is there a specific place in `api` that I should place the new file? Would you prefer for me to place it in `events` with `BlastBlockModifyEvent`, or should I create a new package for it?